### PR TITLE
🎉 New Source: Airship

### PR DESCRIPTION
**Pull request by MatthieuMoissonFranckhauser**
## What
Add new data source connector

## How
Create connector with yaml template

## Recommended reading order
?
## 🚨 User Impact 🚨
No breaking change

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details><summary><strong>Connector Generator</strong></summary>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*

</details>

### CODE REVIEW

**Comment by sajarin** [comment]
Hi @MatthieuMoissonFranckhauser, thanks for the PR. This PR is part Airbyte's Community Maintainer program and someone from our community will be assigned to review your PR and help it get merged. Thanks for being patient and thanks for building this connector.

**Comment by sajarin** [comment]
assigned to @Nmaxime to start the review on 9/17

File: airbyte-integrations/connectors/source-airship/source_airship/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+documentationUrl: https://docs.airbyte.com/category/sources/airship
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Airship Spec
+  type: object
+  required:
+    - access_key
+  additionalProperties: true
+  properties:
+    access_key:
```
Comment (Nmaxime) [inline airbyte-integrations/connectors/source-airship/source_airship/spec.yaml:10]: You can order the inputs (example [here](https://github.com/airbytehq/airbyte/pull/15880/files#diff-a8620b4d2a253892e3f1864be548549e6671e1ebcc348472165051761461e684R17))

File: airbyte-integrations/connectors/source-airship/source_airship/spec.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  title: Airship Spec
+  type: object
+  required:
+    - access_key
+  additionalProperties: true
+  properties:
+    access_key: 
+      type: string
+      description: >-
+        Your API access key
```
Comment (Nmaxime) [inline airbyte-integrations/connectors/source-airship/source_airship/spec.yaml:13]: Please add a link to API access key page if it's possible: [example](https://github.com/airbytehq/airbyte/pull/15880/files#diff-a8620b4d2a253892e3f1864be548549e6671e1ebcc348472165051761461e684R13).
You might also indicate it's a bearer token, the API offers other ways to authenticate.

**Review by Nmaxime — CHANGES_REQUESTED** [review]
Thanks for the contribution @MatthieuMoissonFranckhauser !

Could you add Airship [here](https://github.com/airbytehq/airbyte/tree/master/docs/integrations/sources), in the SUMMARY.md ([doc](https://docs.airbyte.com/connector-development/tutorials/building-a-python-source/#step-12-add-docs)) and [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/builds.md) please ?

A CLA signed and a screenshot of the acceptance tests results are also needed to merge the PR.

Also, this step has to be executed [step](https://docs.airbyte.com/connector-development/tutorials/building-a-python-source/#step-11-add-the-connector-to-the-apiui).

**Comment by sajarin** [comment]
@MatthieuMoissonFranckhauser can you sign the CLA agreement as well as address some of the requested changes? Let us know when you'll be able to do so!

**Comment by sajarin** [comment]
@MatthieuMoissonFranckhauser, could you give us an eta or update on where we are at with this PR? We'd like to get this source merged. Let us know if you're running into any issues!

**Comment by sajarin** [comment]
@MatthieuMoissonFranckhauser going to close this due to inactivity. Please reply and respond to some of the changes here so that we can proceed with the review/merge.

**Pull request closed without merging**